metricbeat/docs/fields.asciidoc

```diff
@@ -42706,6 +42733,43 @@ type: long
 Event loop delay in milliseconds
 
 
+type: scaled_float
+
+--
+
+[float]
+=== event_loop_utilization
```

Comment:
The kibana module can either route data to `metricbeat-*` or `.monitoring-kibana-8-mb` depending on the `xpack.enabled` value provided. The `monitoring-*` template is defined [in elasticsearch](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/resources/monitoring-es-mb.json), could you also add the mappings there ?

Reply:
Sure thing - opened PR here: https://github.com/elastic/elasticsearch/pull/95388